pysmt/environment.py

```diff
@@ -74,6 +74,7 @@ def __init__(self):
         self._factory = None
         # Configurations
         self.enable_infix_notation = False
+        self.enable_div_by_0 = True
```

Comment:
This flag has a strange name: if `enable_div_by_0` is True, we allow the div by zero and we throw a warning, if it is False, we allow the div by zero and we do not raise the warning...

Maybe we can call it `warn_if_div_by_zero`?

Reply:
The idea of the configuration flag was mostly to disable this feature. If this is set to false, there would not be any special handling of the division by zero, and thus we would throw a DivisionByZero exception.

I am also ok with always allowing this behavior and only control whether we throw a warning or not (change this to `warn_if_div_by_zero`.